### DISCUSSION TITLE
Forums for social groups ??

### DISCUSSION THREAD

**Discussion by johnhenley** [DNNCommunity/Dnn.CommunityForums#349] [Q&A]
Most of our recent efforts and releases have focused on improving the internal workings of the Community Forums module and adding features for "traditional" forums use cases. However, there is a "toggle switch" that lets you install a forums instance and use it in combination with a DNN social group. Are you using that feature? Are there improvements you need?

**Comment by WillStrohl** [comment]
I personally know of clients that might be interested in using that.  🤔

**Comment by johnhenley** [comment]
That was actually one nice feature way back when in the days of Active Social. When you created a social group, there was a toggle to create a forum for that group. When DNN acquired Active Modules and folded in social groups, that went away I guess. 

Now (I think) you have to add the forums module to the tab/page you created for the group and in the forums module settings, change the "mode" option.

**Reply by WillStrohl** [reply]
Active Social and Social Groups still have all of their roots in DNN's existing social features.  They just don't get the love they need.

**Comment by Timo-Breumelhof** [comment]
It's still working, I tested it for the client @WillStrohl mentioned.
